platform_versioned_docs/version-23.4/getting-started/rnaseq.mdx

```diff
@@ -0,0 +1,749 @@
+
+:::note 
+Fusion requires a license for use in Seqera Platform compute environments or directly in Nextflow. Fusion can be trialed at no cost. [Contact Seqera](https://seqera.io/contact-us/) for more details.
+:::
+
+### Create compute environment
+
+![Add Platform compute environment](./_images/create-ce.gif)
+
+From the **Compute Environments** tab in your organization workspace, select **Add compute environment** and complete the following fields:
+
+| **Field**                             | **Description**                                            |
+|---------------------------------------|------------------------------------------------------------|
+| **Name**                              | A unique name for the compute environment.                 |
+| **Platform**                          | AWS Batch                                                  |
+| **Credentials**                       | Select existing credentials, or **+** to create new credentials:|
+| **Access Key**                        | AWS access key ID.                                         |
+| **Secret Key**                        | AWS secret access key.                                     |
+| **Region**                            | The target execution region.                               |
+| **Pipeline work directory**           | An S3 bucket path in the same execution region.            |
+| **Enable Wave Containers**            | Use the Wave containers service to provision containers.   |
+| **Enable Fusion v2**                  | Access your S3-hosted data via the Fusion v2 file system.  |
+| **Enable fast instance storage**      | Use NVMe instance storage to speed up I/O and disk access. Requires Fusion v2.|
+| **Config Mode**                       | Batch Forge                                                |
+| **Provisioning Model**                | Choose between Spot and On-demand instances.               |
+| **Max CPUs**                          | Sensible values for production use range between 2000 and 5000.|
+| **Enable Fargate for head job**       | Run the Nextflow head job using the Fargate container service to speed up pipeline launch. Requires Fusion v2.|
+| **Allowed S3 buckets**                | Additional S3 buckets or paths to be granted read-write permission for this compute environment. Add data paths to be mounted in your data studio here, if different from your pipeline work directory.|
+| **Resource labels**                   | `name=value` pairs to tag the AWS resources created by this compute environment.|
+
+
+## Add pipeline to Platform 
+
+:::info
+The [nf-core/rnaseq](https://github.com/nf-core/rnaseq) pipeline is a highly configurable and robust workflow designed to analyze RNA-Seq data. It performs quality control, alignment and quantification.
+
+![nf-core/rnaseq subway map](./_images/nf-core-rnaseq_metro_map_grey_static.svg)
```

Comment:
Looking at this again, the image is fine, I just can't read the small text under Methods. I don't think there is much we can do when it's been shrunk to fit in the box.